src/pro/sky/java/course1/Employee.java

```diff
@@ -0,0 +1,55 @@
+package pro.sky.java.course1;
+
+public class Employee {
+    static int cnt = 0;
+    private int id;
+    private String fio;
+    private int oldel;
+    private double  salary;
+
+    private int genId(){
+        cnt++;
+        return cnt;
```

Comment:
для префиксной записи инкремента (сначала ++, затем возврат значения, у постфиксного наоборот) данный код можно сократить:
```suggestion
        return ++cnt;
```